src/includes/class-recommender-api.php

```diff
@@ -107,6 +110,13 @@ public static function get_instance()
 	 */
 	public function send_event($data, $event_type, $timeout = 5000)
 	{
+        //WP internal logging for incoming events. TODO: Once we have tested sending events to the API properly, delete this.
+        // Needs in wp-config.php
+        // define('WP_DEBUG', true);
+        // define('WP_DEBUG_LOG', true);
+        // The log will be in wp-content
+        error_log($event_type);
+        error_log(print_r($data,true));
```

Comment:
Can we log these with our logger as well? We can then use our own log for testing purposes.